salt/modules/virt.py

```diff
@@ -463,6 +464,7 @@ def _get_disks(dom):
     Get domain disks from a libvirt domain object.
     '''
     disks = {}
+    hypervisor = __salt__['config.get']('libvirt:hypervisor', 'kvm')
```

Comment:
don't use the ``libvirt:hypervisor`` configuration: you should rather use the domain type. I have a pending commit here that removes the use of minidom though the whole module, but you can temporarily do it with minidom and I'll rebase my change later on.

Note that ``libvirt:hypervisor`` is now deprecated.

Reply:
Ack.